### ISSUE TITLE
new next 13 api route

### ISSUE THREAD

**Issue by y-nk** [nextauthjs/next-auth#7085]
### Question 💬

From Next's docs, we're supposed to export GET, POST etc methods as handlers. I was looking at the docs from auth.js and it seems like solid version is actually doing that.

Would there be any way that the api routes of solid could be used with the ui parts of Next? Maybe if we opt-in for customized login/logout pages, then there would be no import/dependency to solidjs?

### How to reproduce ☕️

.

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
Duplicate of #6792

**Comment by y-nk** [comment]
@balazsorban44 no it's not a duplicate. **i'm not asking for resolution, i'm asking if in the meantime the solid version could be used.**

**Comment by balazsorban44** [comment]
In my opinion, you were asking for a resolution, you just got a different one than expected.

It does not make sense to use the Solid package for Next.js and sounds like a bad official recommendation.

You are free to do so in user-land, but this will not be endorsed. As linked above, there is already an issue opened for properly supporting Route Handlers.

**Comment by y-nk** [comment]
> In my opinion, you were asking for a resolution, you just got a different one than expected.

your opinion does make an OP statement true. i used the "question" thing and this answer could have been it, but "duplicate" is not one. It may seem crazy but i know how to use the search and found that thread before, so you could have equally assume that if i were to create a question despite the thread, i could have expected not to be redirected to it.

> It does not make sense to use the Solid package for Next.js and sounds like a bad official recommendation.

I understand that and i do not seek for a "stamped official" recommendation.

> You are free to do so in user-land, but this will not be endorsed. As linked above, there is already an issue opened for properly supporting Route Handlers.

I don't mind this being endorsed or not, i do not seek validation of next-auth in my personal code, rather i asked friendly advice as if it seems doable or not.

> As explained (#6792 (comment)) already, the open PR is blocked by an upstream (Next.js) bug which is being worked on already.

This is also why i searched for alternative meanwhile, as we don't know much how long the upstream will prevent the PR from being merged. I understand it's not next-auth fault but being dependent is always a tough position where you constantly weight "waiting for an upstream resolution" and/or "try differently to avoid blocking an entire community"

> Feel free to not ping on unrelated issues to get more attention in the future.

This is harsh way of treating a community @balazsorban44 - i remained civil in all exchanges, used all the tooling and made sure i provided all informations i could. in return all i got is a plain cold redirect to a vaguely matching topic.
